py3-numpy.yaml

```diff
@@ -34,22 +41,21 @@ pipeline:
       expected-commit: 9815c16f449e12915ef35a8255329ba26dacd5c0
       recurse-submodules: true
 
-  - runs: |
-      python3 setup.py install --prefix=/usr --root="${{targets.destdir}}"
-
-  - uses: strip
-
 subpackages:
-  - name: f2py
-    description: "f2py for numpy (for python3)"
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}} installed for python${{range.key}}
     pipeline:
-      - runs: |
-          mkdir -p "${{targets.subpkgdir}}"/usr/bin
-          mv "${{targets.destdir}}"/usr/bin/f2py* "${{targets.subpkgdir}}"/usr/bin
+      - runs: python${{range.key}} -m pip install . "--root=${{targets.subpkgdir}}" --prefix=/usr
+      - uses: strip
     dependencies:
       runtime:
-        - python3-dev
+        - py${{range.key}}-${{vars.pypi-package}}
+      provides:
+        - py3-${{vars.pypi-package}}
         - numpy
+        - f2py
+      provider-priority: ${{range.value}}
```

Comment:
@smoser note not co-installable as all of them provide f2py.

Reply:
For anything that would be installed as a build dependency we need to make at least portions of it coinstallable?